### PR TITLE
Added year_report endpoint

### PR DESCRIPTION
**Pull request by smmbll**

### CODE REVIEW

File: umibukela/healthe/views.py [hunk 1]
```diff
@@ -67,3 +67,18 @@ def stats(request):
...
     return JsonResponse(stockout_stats(start, end, rows))
+
+@cache_page(3600)
+def year_report(request):
+    """ Provide full year reports
+    for http:s//www.health-e.org.za/medicine-stockouts/ page
+    """
+    year = request.GET.get('year')
+    start_date = arrow.get('%s-01-01' % year).date()
+    end_date = arrow.get('%s-12-31' % year).date()
```
Comment (longhotsummer) [inline umibukela/healthe/views.py]: We need to be careful to honour the agreement Health-E has with the dept not to publicly release data until a month after it has been captured. `end_date` should be no later than 4 weeks before today. eg.

```python
cutoff = date.today() - timedelta(weeks=4)
if end_date > cutoff:
    end_date = cutoff
```

Comment (longhotsummer) [inline umibukela/healthe/views.py:78]: We need to be careful to honour the agreement Health-E has with the dept not to publicly release data until a month after it has been captured. `end_date` should be no later than 4 weeks before today. eg.

```python
cutoff = date.today() - timedelta(weeks=4)
if end_date > cutoff:
    end_date = cutoff
```

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Looks good, just the minor change.

File: umibukela/healthe/views.py (hunk 1 above)
Comment (longhotsummer) [reply umibukela/healthe/views.py:78]: Please sanity check my logic here, it has been a long day :)

Comment (smmbll) [reply umibukela/healthe/views.py:78]: is it a calendar month, i.e. 30 days, or four weeks? i've altered it to reflect your suggestion but wanted to be sure.

**Review by longhotsummer — APPROVED** [review]

**Comment by longhotsummer** [comment]
looks good!

**Pull request merged**